tdrs-frontend/src/components/EditProfile/EditProfile.jsx

```diff
@@ -49,6 +49,7 @@ export const validation = (fieldName, fieldValue) => {
  * There is an input for first and last name and a combo box to select
```

Comment:
@riatzukiza was the work in this file done for this branch, or is there another branch we have pulled in here?

Reply:
This is all from this branch